web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1078,24 +1078,31 @@
                   var vectorFormat = new ol.format.WFS();
               }
 
+
+              //check if proxy is needed
```

Comment:
If the GetCapabilities needed the proxy, use the proxy to GetFeature

Reply:
This code is duplicated from the proxy interceptor (as well as other places...). I don't think this is a good thing, esp. considering how many assumptions are done here.

Also, I'm not sure what are the use cases for proxying requests to a WFS service?

Reply:
I agree, proxying a wfs-get-feature request (to fix mixed content or missing CORS) is not optimal, it should be a conscious decision of the administrator to allow this (not the default setting).